src/scrabble.js

```diff
@@ -1,5 +1,172 @@
-function scrabble() {
+function scrabble(word) {
+  if (typeof word !== 'string') {
+    return 0
+  }
 
+  const caseSensitivity = word.toUpperCase()
+  let charExtract
+  let charPoints
+  let scrabbleScore = 0
+  let doublePoints = 0
+  let triplePoints = 0
+  const incorrectTokenControl = () => {
+    const acceptableChars = [
+      'A',
+      'B',
+      'C',
+      'D',
+      'E',
+      'F',
+      'G',
+      'H',
+      'I',
+      'J',
+      'K',
+      'L',
+      'M',
+      'N',
+      'O',
+      'P',
+      'Q',
+      'R',
+      'S',
+      'T',
+      'U',
+      'V',
+      'W',
+      'X',
+      'Y',
+      'Z',
+      '{',
+      '}',
+      '[',
+      ']',
+      ' ',
+      '',
+      '.'
+    ]
+    return acceptableChars.filter((pass) => pass.includes(charExtract))
+  }
```

Comment:
I like the use of filter to extract just the valid characters from the input string.